pom.xml

```
@@ -48,14 +48,14 @@
     <maven.wagon.ssh.external.version>2.7</maven.wagon.ssh.external.version>
     <apache.rat.plugin.version>0.13</apache.rat.plugin.version>
 
-    <citrus.version>3.4.0</citrus.version>
+    <citrus.version>4.3.0-SNAPSHOT</citrus.version>
```

```suggestion
    <citrus.version>4.3.1</citrus.version>
```

this however will have to be `4.3.1` before merging!